src/util/color.rs

```diff
@@ -60,3 +63,17 @@ impl Default for ColorChoice {
         Self::Auto
     }
 }
+
+impl ValueEnum for ColorChoice {
+    fn value_variants<'a>() -> &'a [Self] {
+        &[Self::Auto, Self::Always, Self::Never]
+    }
+
+    fn to_possible_value(&self) -> Option<PossibleValue> {
+        Some(match self {
+            Self::Auto => PossibleValue::new("auto").help("Enables colored output only when the output is going to a terminal/TTY"),
+            Self::Always => PossibleValue::new("always").help("Enables colored output regardless of whether or not the output is going to a terminal/TTY"),
+            Self::Never => PossibleValue::new("never").help("Disables colored output regardless of whether or not the output is going to a terminal/TTY"),
```

Comment:
Feel like these are a bit verbose particularly in the context they will be displayed and it will bury the important part of the information.

Reply:
That's fair. I can try and come up with something more terse.